### PR TITLE
add subclass axiom to cardio terms (with expert reviewer ID)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #6740

1. Add subclass axioms for **stay** terms
FYI: ROBOT command for the **stay** spreadsheet: 
`sh run.sh make TEMPLATE_URL="https://docs.google.com/spreadsheets/d/e/2PACX-1vTGBAMfHVn67fGn9IL2Noj7kMTYovqKJPPemq939uLdyT--FvAc2_QpKjh23yhtrw/pub?gid=609591374&single=true&output=tsv" merge_template -B`

2. add comments for **leave** terms
ROBOT command for the **leave** spreadsheet:
`sh run.sh make TEMPLATE_URL="https://docs.google.com/spreadsheets/d/e/2PACX-1vTGBAMfHVn67fGn9IL2Noj7kMTYovqKJPPemq939uLdyT--FvAc2_QpKjh23yhtrw/pub?gid=1240010881&single=true&output=tsv" merge_template -B`

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -127660,6 +127660,7 @@ xref: MESH:C563239 {source="MONDO:equivalentTo"}
 xref: OMIM:105805 {source="Orphanet:99092", source="MONDO:equivalentTo", source="Orphanet:99092/e"}
 xref: Orphanet:99092 {source="MONDO:equivalentTo", source="OMIM:105805"}
 xref: UMLS:C1387721 {source="Orphanet:99092", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:105805"}
+is_a: MONDO:0004995 {source="https://orcid.org/0000-0001-6718-3559", source="https://orcid.org/0000-0001-9310-0163"} ! cardiovascular disorder
 is_a: MONDO:0019512 {source="https://orcid.org/0000-0002-3458-4839"} ! congenital heart malformation
 is_a: MONDO:0021147 {source="https://orcid.org/0000-0002-4142-7153"} ! disorder of development or morphogenesis
 property_value: confidence "7.125" xsd:double
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:127666]: see below- the GitHub URL has already been added, so it's not being added with the [ROBOT template](https://docs.google.com/spreadsheets/d/1fIqp9B434HjNWZmjqWKYXjvDY6e3LsTg/edit#gid=609591374)

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
I am having some issue with GitHub right now, so I cannot check in protege, but: 
1) did you merge the template in tab "ROBOT Reviewer-Stay Curator-Leave" ?
2) The last step is to review spreadsheet called "Reviewer-Leave Curator-Stay" and see whether these terms should be in the branch or not (and then do the appropriate changes)

**Comment by nicolevasilevsky** [comment]
> did you merge the template in tab "ROBOT Reviewer-Stay Curator-Leave" ?

Sorry, I missed that one, I'll merge it

**Comment by nicolevasilevsky** [comment]
> The last step is to review spreadsheet called "Reviewer-Leave Curator-Stay" and see whether these terms should be in the branch or not (and then do the appropriate changes)

will do!

**Comment by nicolevasilevsky** [comment]
@sabrinatoro everything should be done now.

**Review by sabrinatoro — APPROVED** [review]
Review: 
- [x] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [x] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab)
- [x] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [x] remove parents in the cardiovascular branch (see "Reviewer-Leave Curator-Stay" Tab )
- [x] add new comments (see "ROBOT Reviewer-Leave Curator-Stay" tab)

All the items on the to-do list were done in the PR, and there no unexpected changes.
Thank you @nicolevasilevsky, this is ready to merge!

**Pull request merged**